tests/package.json

```
@@ -5,8 +5,8 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "test": "cypress run",
-    "test:open": "cypress open"
+    "test": "export TESTING_USERNAME=alex && export TESTING_PASSWORD=Alex1234 && cypress run",
+    "test:open": "export TESTING_USERNAME=alex && export TESTING_PASSWORD=Alex1234 && cypress open"
```

```suggestion
    "test": "TESTING_USERNAME=alex TESTING_PASSWORD=Alex1234 cypress run",
    "test:open": "TESTING_USERNAME=alex TESTING_PASSWORD=Alex1234 cypress open"
```
You can use this syntax instead to assign environments to single command.